src/locales/default.json

```diff
@@ -587,6 +587,11 @@
       "label": "Anyone can vote",
       "description": "Anyone with voting power can cast a vote."
     },
+    "basic": {
```

Comment:
@zzuziak please review newly added content

Reply:
@samuveth Can we ignore translations for these validations, in future we may get so many voting validations, it can be hard to maintain this here. maybe we can add `label` and `description` from strategies repo itself?

Reply:
Comments look good to me :ok_hand:

Reply:
@ChaituVR yes, feel free to make the changes on then I'll integrate it on Ui.